### DISCUSSION TITLE
Historie Problem

### DISCUSSION THREAD

**Discussion by FoggySolar** [solectrus/solectrus#3258] [General]
Hallo,
ich habe erneut und ohne das an meinem Solectrus Docker auf meiner Synology Docker Installation etwas gemacht wurde das Problem, dass auf einmal in meiner Historie aktuell die Daten aus diesem März , eine komplette Woche im Juni und alles was vor Juli 2023 war weg sind.
Vor einer Woche, als ich zuletzt geschaut hatte war definitiv alles noch in Ordnung und alle Daten da. 
Die csv Dateien sind alle vorhanden und dort gab es auch keinerlei Änderung von mir.
Kann ich irgendwie die vorhandenen Daten (csv Dateien), ohne den Docker noch einmal neu zu installieren, überprüfen und ggf. neu schreiben lassen?
Vielen Dank im Voraus für eure Hilfe.

**Comment by ledermann** [comment]
Das ist sehr ungewöhnlich, mir fallen folgende potenzielle Erklärungen ein:

- Deine Konfiguration in `.env` verweist auf falsche Fields oder Measurements. Das heißt, ein Kollektor schreibt Daten in InfluxDB nach X, SOLECTRUS selbst liest aber in Y. 
- In den betreffenden Zeiträumen war ein Kollektor außer Betrieb und hat nichts geschrieben

Ich würde versuchen, mit dem UI der InfluxDB die Daten genauer zu analysieren.

Ja, du kannst die Daten jederzeitig neu über den CSV-Importer einlesen. Das ist unschädlich, auch wenn teilweise bereits Daten vorhanden sind.

Nach dem Import nicht vergessen, den Cache zu löschen: `docker exec -it solectrus-redis-1 redis-cli FLUSHALL`

**Reply by FoggySolar** [reply]
Hallo @ledermann ,
Danke für deine schnelle Antwort.
Eben habe ich versucht die Daten noch einmal einzulesen, leider ohne Erfolg.
Ich werde mal schauen ob ich in der InfluxDB etwas finde.
Wer weiß was da los ist, ggf. mache ich noch einmal eine Neuinstallation.
Vielen Dank auch für die neue Version 
Vielleicht kann ich dabei deinen Konfigurator gleich mal testen ;)

**Reply by ledermann** [reply]
Hast du wirklich den Redis-Cache gelöscht? Das wird gerne vergessen und sorgt dafür, dass man nach einem Import keinerlei Änderungen sieht.

Ansonsten sieht es nach einem Measurement/Field-Durcheinander aus. Das könntest du über die UI der InfluxDB herausbekommen. Der CSV-Importer muss an die Stelle schreiben, wo SOLECTRUS dann auch lesen wird. Prüfe insbesondere das Measurement!

**Reply by FoggySolar** [reply]
Ja, den REDIS Cache hatte ich gelöscht.
Mal schauen, so bewandert in der InfluxDB bin ich auch noch nicht.
Vielleicht bin ich schneller mit einer fixen Neuinstallation des Dockers ;)
